stories/example-screen-staff-list.stories.js

```diff
@@ -289,322 +289,105 @@ const tabsDoubleRowHTML = `
       </a>
     </li>
   </ul>
-  <ul class="nav nav-tabs bg-white border border-light border-top-0 ps-7 pe-7">
-    <li class="nav-item">
-      <a class="nav-link active fs-5" href="#">
+
+  <ul class="nav nav-tabs bg-white ps-7 pe-7">
+    <li class="nav-item py-2 px-3">
+      <a class="fs-6 text-primary" href="#" style="font-weight: 400;">
         <span>All Jobs</span>
       </a>
     </li>
-    <li class="nav-item">
-      <a class="nav-link fs-5" aria-current="page" href="#">
+    <li class="nav-item p-2">
+      <a class="fs-6 text-muted" href="#" style="font-weight: 400;">
         <span>Recommended Jobs</span>
       </a>
     </li>
-    <li class="nav-item">
-      <a class="nav-link fs-5" href="#">
+    <li class="nav-item p-2">
+      <a class="fs-6 text-muted" href="#" style="font-weight: 400;">
         <span>My Favorites</span>
       </a>
     </li>
-    <li class="nav-item">
-      <a class="nav-link fs-5" href="#">
+    <li class="nav-item p-2">
+      <a class="fs-6 text-muted" href="#" style="font-weight: 400;">
         <span>Applications</span>
       </a>
     </li>
   </ul>
 `;
 
+const batchAndPagination = `
+  <div class="d-flex align-items-center justify-content-between flex-wrap mx-3 mt-4 mb-3">
+    <div class="d-flex align-items-center justify-content-center">
+      <select class="form-select fs-6" aria-label="Default select example" style="padding-right: 2rem; line-height: 1.3;">
+        <option selected>Batch Options</option>
+        <option value="1">One</option>
+        <option value="2">Two</option>
+        <option value="3">Three</option>
+      </select>
+      <span class="fs-6 text-muted ms-3" style="min-width: 120px;">1 item selected</span>
+    </div>
+    <div class="d-flex align-items-center flex-wrap">
+      <div class="w-auto me-4">
+        <select class="form-select shadow-none border-0 fs-6" aria-label="Default select example" style="line-height: 1.3;">
+          <option value="20">Show 20</option>
+          <option value="50">Show 50</option>
+          <option value="100">Show 100</option>
+        </select>
+      </div>
+      <div class="d-flex">
+        <button type="button" class="btn btn-action btn-action-icon-only" style="border: 1px solid #aaaaaa; border-top-right-radius: 0; border-bottom-right-radius: 0;">
+          <svg class="icon-lg-font-size mt-n1 text-danger">
+            <use xlink:href="${iconSprite}#tabler-chevron-left" />
+            <span class="visually-hidden">Previous</span>
+          </svg>
+        </button>
+        <div class="d-flex mx-0">
+          <select class="form-select shadow-none fs-6 pr-5 rounded-0 border-end-0 border-start-0" aria-label="Default select example" style="padding-right: 2rem; line-height: 1.3;">
```

Comment:
Remove inline styles and replace with classes, `pe-7` and `lh-sm` (or whatever works).